src/layers/conv.jl

```diff
@@ -506,3 +506,44 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", AGGR2STR[l.aggr])
     print(io, ")")
 end
+
+
+"""
+    GINConv([graph, ] nn, eps, train_eps
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::R
+    train_eps::Bool
```

Comment:
Maybe we can rethink the need of `train_eps` here. Does it worth as a separate argument to control a trainable `eps` or not?

Reply:
For example, if we set it as `0.`, it may means it is not trainable.

Reply:
Hmm, I guess the default behavior should be that eps is not trainable. I'll leave the implementation of untrainable parameters but remove the argument (for now).

Reply:
Maybe there are some hints hidden in the implementation of pytorch geometric. We could dig it out and have some discussions.